packages/@azure/storage/blob/package-lock.json

```diff
@@ -6468,7 +6468,7 @@
     },
     "sprintf-js": {
       "version": "1.0.3",
-      "resolved": "https://registry.npmjs.org/sprintf-js/-/sprintf-js-1.0.3.tgz",
```

Comment:
Please remove this change from this PR, and create a separate PR with this change if we want to merge it.  I'd rather keep package-lock.json updates in their own PRs (or in the PRs of associated changes).

Reply:
Well, this was unintentional!